spm-monitor/src/test/java/com/sematext/spm/client/command/CommandPollingTest.java

```diff
@@ -364,7 +364,7 @@ public void run() {
 
     server.pendingCommands = Arrays.asList(cancelC1);
 
-    TCommandResponse response = server.responses.poll(5, TimeUnit.SECONDS);
+    TCommandResponse response = server.responses.poll(10, TimeUnit.SECONDS);
```

Comment:
Now that we have ThriftUtils in another module - is 10 seconds timeout needed, or would 5 be enough?

Reply:
5 should be plenty, changing it back